src/Gui/TaskView/TaskImage.cpp

```diff
@@ -48,38 +48,65 @@
 #include <Gui/ViewProviderDocumentObject.h>
 #include <Gui/TaskView/TaskView.h>
 
-#include "TaskImageScale.h"
-#include "ui_TaskImageScale.h"
+#include "TaskImage.h"
+#include "ui_TaskImage.h"
 
 
 using namespace Gui;
 
-/* TRANSLATOR Gui::TaskImageScale */
+/* TRANSLATOR Gui::TaskImage */
 
-TaskImageScale::TaskImageScale(Image::ImagePlane* obj, QWidget* parent)
+TaskImage::TaskImage(Image::ImagePlane* obj, QWidget* parent)
   : QWidget(parent)
-  , ui(new Ui_TaskImageScale)
+  , ui(new Ui_TaskImage)
   , feature(obj)
-  , aspectRatio{1.0}
+  , aspectRatio(1.0)
 {
     ui->setupUi(this);
     ui->pushButtonCancel->hide();
-    ui->spinBoxWidth->setValue(obj->getXSizeInPixel());
-    ui->spinBoxHeight->setValue(obj->getYSizeInPixel());
+
```

Comment:
The constructor is fairly big and it's best to move parts of the code here to some functions:
* The initialization of the sliderTransparency can be moved to its own function
* The calls of connect() can be moved to its own function